test/nodes/LatchedSequence.test.ts

```diff
@@ -58,36 +61,32 @@ describe('LatchedSelector', function() {
 		const botState = new RobotState();
 		botState.laserCooldownTime = 0;
 
-		const p = shutdownSequence.handleEvent(botState, 'lowBattery');
+		const res = shutdownSequence.handleEvent(botState, 'lowBattery');
 
-		return p.then((res) => {
-			assert.equal(res, rc.SUCCESS, 'Behavior Tree success');
-			assert.equal(botState.commands.length, 3, 'Need Three Commands');
-			assert.equal(botState.commands[0], 'motorsStopped');
-			assert.equal(botState.commands[1], 'lasersCooled');
-			assert.equal(botState.commands[2], 'powerOff');
-		});
+		assert.equal(res, rc.SUCCESS, 'Behavior Tree success');
```

Comment:
might want to globally find/replace `assert.equal` with `assert.strictEqual` /shrug